From 447c68986f29ba8b832a5844f7e10ed1c94713dc Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 3 May 2023 20:49:24 +0100
Subject: [PATCH 1/1] core/context: Exclude root from admin users

root is well, root. It doesn't need to be added to special groups, right?
---
 core/context.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/context.go b/core/context.go
index e419848..7fa1dff 100644
--- a/core/context.go
+++ b/core/context.go
@@ -77,7 +77,7 @@ func (c *Context) FilterUsers(filters ...string) (filtered []User) {
 			fallthrough
 		case !it.IsActive && contains(filters, "system"):
 			fallthrough
-		case (it.IsRoot || it.IsAdmin) && contains(filters, "admin"):
+		case it.IsAdmin && contains(filters, "admin"):
 			filtered = append(filtered, it)
 		}
 	}
-- 
2.40.1

